int32 traj_type     #0: Test Path,   1: Minco Path

nav_msgs/Path nav_path

kinematics_simulator/MincoTraj minco_path
